src/util/pg.py

```diff
@@ -411,14 +429,20 @@ def pg_text2html(s, wrap="p"):
                     '&Tab;'),
                 '{closing_tag}')
          END
-    """.format(
-        opening_tag="<{}>".format(wrap) if wrap else "",
-        closing_tag="</{}>".format(wrap) if wrap else "",
-        src=s,
-        esc=pg_html_escape(s, quote=False),
+        """.format(
+            opening_tag="<{}>".format(wrap) if wrap else "",
+            closing_tag="</{}>".format(wrap) if wrap else "",
+            src=s,
+            esc=pg_html_escape(s, quote=False),
+        )
     )
 
 
+def get_value_or_en_translation(cr, table, column):
```

Comment:
Nice to have: this should had the `pg_` prefix... but maybe it isn't worth renaming it and keeping the old name as alias.